Reference/Searching/Examine/pdfindex-multisearcher.md

```diff
@@ -1,75 +1,41 @@
----
-versionFrom: 8.0.0
-versionTo: 8.0.0
+﻿---
+versionFrom: 9.0.0
+versionTo: 10.0.0
 ---
 
 # PDF indexes and multisearchers
 
-:::note
-This document has been verified for Umbraco 8.
-
-If you are using Umbraco 9 or later versions, please refer to the note on the [Examine documentation landing page](index.md) for more details.
-:::
-
 If you want to index PDF files and search for them you will need to use the [UmbracoExamine.Pdf extension package](https://github.com/umbraco/UmbracoExamine.PDF).
 
 ## Installation
 
 Install with Nuget:
-`Install-Package UmbracoCms.UmbracoExamine.PDF`
+`dotnet add package Umbraco.ExaminePDF`
 
 You will then have a new Examine index called "PDFIndex" available.
 
 ## Multi index searchers
 
```

Comment:
Are multi index searchers explained elsewhere? Otherwise it might be good with a little intro to what they are before we dive right in.